enarx-keep/src/main.rs

```diff
@@ -14,6 +14,8 @@ use binary::Component;
 use anyhow::Result;
 use structopt::StructOpt;
 
+use std::path::PathBuf;
```

Comment:
If we're just going to be using a UnixListener (which I suspect we are), there's no need for an explicit path.   You can create a new one with a cast String (see https://github.com/enarx/enarx/pull/802/files#diff-4ceb5185f3d05238cb8d7553500d56f5)
https://doc.rust-lang.org/beta/std/os/unix/net/struct.UnixListener.html#method.bind

Reply:
@MikeCamel We aren't using a `UnixListener`. We're doing `UnixStream::connect()` which takes a path. `enarx-keepmgr` should create the `UnixListener` and `enarx-keep` should receive the address to the socket it should connect to. All details of the protocol over that socket are backend specific.